src/_includes/training_hero.liquid

```diff
@@ -0,0 +1,24 @@
+<section class="training-hero">
+    <div class="training-hero__background-image" style="background-image: url({{ site.baseurl }}{{ page.intro.background-image }})"> </div>
+    <div class="training-hero__lower-background-block"></div>
+    <div class="training-hero__content-container">
+      <div class="training-hero__course-content-info-container">
+        <h1 class="training-hero__title">Crafting Code</h1>
+        <span class="training-hero__duration">Duration: 2 days</span>
+      </div>
+      <div class="training-hero__course-instructor-container">
+        <span class="training-hero__course-instructor-avatar" style="background-image: url({{ site.baseurl }}{{ page.intro.instructor-avatar }})"></span>
+        <div class="training-hero__course-instructor-info">
+          <span class="training-hero__course-instructor-title">Course Instructor</span>
+          <span class="training-hero__course-instructor">Sandro Mancuso</span>
+        </div>
+      </div>
+      <div class="training-hero__intro-container">
+        <p class="training-hero__intro-description">This course is designed to help developers write code that is well-crafted. Meaning code that is clean,
+        testable, maintainable, and an expression of the business domain. The course is entirely hands-on,
+        designed to teach developers practical techniques they can immediately apply to real-world projects.
+        </p>
+        {% include cta_primary_link.html color='coral' text="Enquire now" %}
+      </div>
+    </div>
+</section>
```

Comment:
In this file, it looks like there are 3 sections. Would each of those sections have its own block name? 
For example, training-hero__course-instructor-* is repeated a few times, could the block be 'training-hero-course-instructor'? ie, training-hero-course-instructor__avatar
Just an observation, not totally sure myself :)